### DISCUSSION TITLE
Linting and more in VS Code based on slang

### DISCUSSION THREAD

**Discussion by racodond** [MikePopoloski/slang#1191] [Show and tell]
Hi Mike, Hi all slang users and contributors,

Thanks to your tremendous work on slang, we just released a new version of Linty. Our linting engine is now based on slang.
Over 200 checks can be performed in real-time in VS Code: https://hdl-designer.linty-services.com or in your continuous integration process (GitHub, GitLab, etc.).

In addition to linting based on slang, Linty also provides checks, reports and graphs on design hierarchy, CDC, RDC and many more.

Feel free to test the free version (150+ checks): https://hdl-designer.linty-services.com/
We'd also be glad to provide licences for the full version.

We would really appreciate your feedback. Please, do not hesitate to contact me if you have any questions or if you'd like to schedule a quick call.

Many thanks again for the great work.

Kind regards,

David Racodon
CTO at Linty Services

**Comment by MikePopoloski** [comment]
Nice, thanks for letting me know. Glad to hear slang has been useful as a base for your product.
